### DISCUSSION TITLE
Duplicate Titles

### DISCUSSION THREAD

**Discussion by unOmni** [lncrawl/lightnovel-crawler#1692] [Ideas]
![Screenshot_20221005-210131_Legado.png](https://user-images.githubusercontent.com/106206117/194171364-c8fc16f8-765a-4ffe-b16d-a8761c246667.png)

Some sites include an extra title within the actual content which is a little annoying for me personally atleast. So, I was wondering if it's possible to remove the secondary title.

**Comment by dipu-bd** [comment]
It is possible. Request it by creating an issue here: https://github.com/dipu-bd/lightnovel-crawler/issues/new?assignees=&labels=source-issue&template=3_source-issue.md&title=Fix+this+source
